app/views/devise/shared/_links.html.haml

```diff
@@ -0,0 +1,19 @@
+- if controller_name != 'sessions'
```

Comment:
if you haven't modified this please remove it

Reply:
also out of scope

Reply:
removed!